src/composables/accounts.ts

```diff
@@ -72,6 +107,17 @@ export const useAccounts = createCustomScopedComposable(() => {
     { backgroundSync: true },
   );
 
+  const arePrivateKeysAccountsRestored = computed(() => (
+    (arePrivateKeysAccountsEncryptedRestored.value && !accountsPrivateKeysEncrypted.value)
+    || (arePrivateKeysAccountsEncryptedRestored.value && arePrivateKeysAccountsDecrypted.value)
+  ));
+
+  const accountsPrivateKeys = computed<IAccountRaw[]>(() => JSON.parse(accountsPrivateKeysDecrypted.value || '[]'));
```

Comment:
As there is a `addPrivateKeyAccount` I suggest to rename this variable to `privateKeyAccountsRaw`. This way it will also match the naming convention of the `accountsRaw` variable.